docs/streams/developer-guide/config-streams.html

```diff
@@ -684,70 +728,64 @@ <h4><a class="toc-backref" href="#id18">Default Values</a><a class="headerlink"
               <td>Consumer</td>
               <td>1000</td>
             </tr>
-            <tr class="row-odd"><td>rocksdb.config.setter</td>
-              <td>Consumer</td>
-              <td>&nbsp;</td>
-            </tr>
             </tbody>
           </table>
         </div>
+        <div class="section" id="parameters-controlled-by-kafka-streams">
+          <h3><a class="toc-backref" href="#id26">Required configuration parameters</a><a class="headerlink" href="#parameters-controlled-by-kafka-streams" title="Permalink to this headline"></a></h3>
```

Comment:
`Required` ? I thought those are the one a user cannot set?

Reply:
D'oh! The perils of copy-paste.